flexn-io/renative
# improve regex overrides

## Description

- Regex overrides improvements (prevent duplication)
- `npx rnv patch reset` => revert applied overrides


## Related issues

- https://github.com/flexn-io/renative/issues/1691

## Npm releases

n/a

## Testing 

- `yarn bootstrap-clean`
- run any platform => overrides were applied ((to make sure you can check node_modules by following the path in the summary)  and  .rnv/overrides folder was created in the root
-  when you run the same command again, overrides should not be applied a second time
- you can change any override and it should change in `.rnv/overrides/applied_overrides.json` and be applied to node_modules.
- run `npx rnv patch reset` => reverts applied overrides + deletes .rnv/overrides folder
